docs/docs-components/buttons/CodeExampleTextDirectionButton.js

```diff
@@ -12,7 +12,8 @@ export default function CodeExampleTextDirectionButton({
   currentTextDirection,
   onClick,
 }: Props): Node {
-  const label = 'Toggle text direction for code example';
+ 
+  const label = `Read contents ${currentTextDirection === 'ltr' ? 'right to left' : 'left to right'} for example`;
```

Comment:
Feel free to discard, I made this change on this branch since it was open, but I can make a new PR

Reply:
Oh, thanks. In the future I suggest leaving that as a comment rather than pushing changes to someone else's PR, unless they asked you to. PR review should be more of discussion and suggesting changes rather than imposing your changes.

Reply:
Sounds good! Makes total sense at proper PR etiquette. I'll avoid from making changes directly (or I'll make a child PR). Thanks!